test/Microsoft.AspNetCore.Authentication.Test/OpenIdConnect/TestDefaultValues.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.IdentityModel.Protocols;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+
+namespace Microsoft.AspNetCore.Authentication.Tests.OpenIdConnect
+{
+    internal class TestDefaultValues
```

Comment:
Nit: This class still seems like an oddball with all statics/constants, maybe it just can be merged with TestServerBuilder